clients/admin-ui/src/features/common/FixedLayout.tsx

```diff
@@ -9,42 +9,49 @@ import { NavTopBar } from "~/features/common/nav/v2/NavTopBar";
 const FixedLayout = ({
   children,
   title,
+  isDefaultLayoutPadding,
 }: {
   children: React.ReactNode;
   title: string;
-}) => (
-  // NOTE: unlike the main Layout, this layout specifies a fixed height
-  // of 100vh for the page, and overflow="auto" for the main content area. This
-  // allows the content area to fitting the *viewport* height, which is a
-  // special case layout needed by the datamap spatial & table views so that we
-  // scroll within those elements instead of scrolling the overall page itself
-  //
-  // Ideally we could have one common layout used by *all* pages, but we couldn't
-  // come up with a common method here without introducing other gotchas, so having
-  // a slightly different layout was decided as the most maintainable option
-  <Flex data-testid={title} direction="column" height="100vh">
-    <Head>
-      <title>Fides Admin UI - {title}</title>
-      <meta name="description" content="Privacy Engineering Platform" />
-      <link rel="icon" href="/favicon.ico" />
-    </Head>
-    <Header />
-    <NavTopBar />
-    <Flex
-      as="main"
-      overflow="auto"
-      flexGrow={1}
-      paddingTop={10}
-      paddingLeft={10}
-      gap={10}
-    >
-      <Box flex={0} flexShrink={0}>
-        <NavSideBar />
-      </Box>
-      <Flex direction="column" flex={1} minWidth={0}>
-        {children}
+  isDefaultLayoutPadding?: boolean;
+}) => {
+  const style = isDefaultLayoutPadding
+    ? {
+        padding: "40px",
+        paddingRight: "48px",
+      }
+    : {
+        paddingTop: "40px",
+        paddingLeft: "40px",
+      };
```

Comment:
This is the actual change. I'm surprised it diffed it like this.

Reply:
hmm why do we want FixedLayout to have an alternate styling? I'd expect FixedLayout to be standard across everything, but there are probably cases not to do that! just trying to figure out if this is a design discrepancy, or if something else is going on.

also, an approach I'd prefer is to pass additional props as a prop into FixedLayout, i.e.

```typescript
const FixedLayout = ({
  children,
  title,
  mainProps,
}: {
  children: React.ReactNode;
  title: string;
  mainProps?: FlexProps
}) => {
...
  <Flex as="main" ... {...mainProps} />
}
```

I think that makes it a bit more flexible, and doesn't require FixedLayout to be responsible for knowing about how other components need to pad

Reply:
It needs different styling because the datamap page (the only other page that uses it AFAIK) is expected to have no right padding and hug the right side of the screen. This new page that uses it is expected to have right padding.

That's a good idea! I'll update it to do it that way.